assets/scss/components/_product.scss

```diff
@@ -1,3 +1,178 @@
 /* ==========================================================================
    #PRODUCT
    ========================================================================== */
+
+.c-product {
+    display: grid;
+    grid-template-columns: 100%;
+    grid-template-areas:
+        'header'
+        'image'
+        'info';
+
+    @include bp(small) {
+        display: grid;
+        column-gap: rem-calc(50px);
+        grid-template-columns: auto 50%;
+        grid-template-rows: min-content 100%;
+        grid-template-areas:
+            'header header'
+            'image info';
+    }
+
+    @include bp(medium) {
+        grid-template-areas:
+            'image header'
+            'image info';
+    }
+
+    .c-grid-list & {
+        display: flex;
+    }
+}
+
+.c-product__title {
+    font-family: $font-semi-bold;
+    font-weight: 600;
+    font-size: rem-calc(42px);
+    line-height: rem-calc(56px);
+}
+
+.c-product__image-wrapper {
+    position: relative;
+    margin: 0;
+}
+
+.c-product__image-link {
+    display: block;
+    aspect-ratio: 616 / 920;
+    height: 100%;
+    width: auto;
+    border-radius: rem-calc(4px);
+}
+
+.c-product__image {
+    width: auto;
+    height: 100%;
+    object-fit: cover;
+    object-position: center center;
+}
+
+.c-product__thumbs {
+    background-color: $color-secondary-50;
+    display: flex;
+    gap: rem-calc(24px);
+    padding: rem-calc(24px);
+    overflow-x: scroll;
+    border-bottom-left-radius: rem-calc(4px);
+    border-bottom-right-radius: rem-calc(4px);
+}
+
+.c-product__thumb {
+    border-radius: rem-calc(4px);
+    display: block;
+    aspect-ratio: 1 / 1.5;
+    width: 100px;
+    height: auto;
+    overflow: hidden;
+    opacity: 0.64;
+
+    &.is-active {
+        opacity: 1;
+    }
+}
+
+.c-product__thumb-image {
+    width: auto;
+    height: 100%;
+    object-fit: cover;
+    object-position: center center;
+}
+
+.c-product__review-link {
+    color: $color-primary-300;
+    font-family: $font-semi-bold;
+    font-weight: 600;
+    font-size: rem-calc(14px);
+    line-height: rem-calc(16px);
+
+    &:hover {
+        color: $color-primary-300;
+    }
+}
+
+.c-product__reviews {
+    display: flex;
+    align-items: center;
+    gap: rem-calc(8px);
+}
+
+.c-product__description {
+    * {
+        color: $color-neutrals-500;
+        font-family: $font-regular;
+        font-weight: 400;
+        font-size: rem-calc(18px);
+        line-height: rem-calc(24px);
+        margin-block: rem-calc(24px);
+    }
+
+    a {
+        color: $color-primary-300;
+    }
+}
+
+.c-product__pricing {
+    display: flex;
+    gap: rem-calc(8px);
+
+    .c-price__without-tax {
```

Comment:
Few things I found not quite right here. 

1. Why can't we have `.c-product__pricing--no-tax` instead of `.c-price__without-tax` ? this `c-price` suggest that it's a separate component which I don't think is a case here
2. It's a bit unclear what `&--non-sale` does in this case. In general it's better to avoid negation in class naming as they may be confusing e.g. `--non-sale` means that the default price is "on sale"? 🤔  Definitelly better approach would be to use `--sale`, than everything is crystal clear :) 
3. I'm not sure why `--non-sale` relates to `.c-price__without-tax` instead of the parent: `.c-product__pricing`?